setup.py

```diff
@@ -51,7 +51,7 @@ def main():
         },
         scripts=[],
         zip_safe=False,
-        install_requires=[],
+        install_requires=['requests', 'PIL'],
```

Comment:
I'm more than happy to have them as optional. Unfortunately I'm not good enough at Python to understand how you would do that in this project. In the other mixin, I've just made it an entirely separate python library and that keeps everything out of this project, but I'm very open to suggestions if you have a good idea of how to do that and still keep this resizer in the lib.

@winhamwr mentioned that resizing might actually be worth including as default behavior, so I'm not sure how that would affect this decision.